### ISSUE TITLE
Constant stream of _agent.destroy is not a function 

### ISSUE THREAD

**Issue by fragosti** [solana-labs/solana-web3.js#1011]
Just upgraded to `0.92.1` and getting constant stream of these errors. Seems to be from here https://github.com/solana-labs/solana-web3.js/blob/6441b4d202b40f260d7502f16f56ac22cae21939/src/agent-manager.js#L39

**Comment by jstarry** [comment]
Thanks, @fragosti what is your project setup like? Are you using Node.js or using a bundler?

**Comment by fragosti** [comment]
We are using https://parceljs.org/typeScript.html for now – so a bundler yes.

**Comment by jstarry** [comment]
@fragosti I just pushed a fix and tested on parcel v2 (should also work on v1). Can you give this a shot:

```
    "@solana/web3.js": "github:jstarry/solana-web3.js#parcel",
```

(Might have to delete parcel cache)

**Comment by jstarry** [comment]
Fix can be found in v0.92.2

**Comment by fragosti** [comment]
@jstarry thanks! sorry lost track of this one. appreciate it
